sql/mssql/inspect.go

```diff
@@ -373,6 +444,29 @@ func nArgs(start, n int) string {
 	return b.String()
 }
 
+// defaultExpr returns the default expression of the given column.
+//
+// https://learn.microsoft.com/en-us/sql/relational-databases/tables/specify-default-values-for-columns
+func (i *inspect) defaultExpr(_ *schema.Column, x string) schema.Expr {
+	// Remove the parenthesis from the expression.
+	x = mayUnwrap(x)
```

Comment:
Why do you remove this if it’s returned like this from the database? Also, sqlx has some code for checking balanced parens, you might want to use it instead.

Reply:
MS-SQL alway wrap with double parenthesis. For example, `6` will become `((6))`, a string `'6'` become `('6')`, and a function `getdate()` become `(getdate())`.

So, unwrap for checking the expression type.